en/docs/learn/export-user-info-to-hubspot.md

```diff
@@ -0,0 +1,37 @@
+#Exporting WSO2 Identity Server User Information to HubSpot
+
+
+HubSpot is a customer relationship management solution that brings companies and customers together. 
+This topic provides instructions on exporting user data from WSO2 Identity Server (WSO2 IS) to the 
+Hub spot platform. 
+
+
+Let's get started!
+
+!!! tip "Before you begin!"
+    Create **users.csv** file by following the instructions in 
+    [Exporting User Data From WSO2 Identity Server](../learn/exporting-user-data-from-wso2-is.md).
+    
+##Try it Out
+
+1. Login into [HubSpot](https://app.hubspot.com/) and got to [contact](https://app.hubspot.com/contacts) section.
```

Comment:
Login --> Log in

got to [contact]() section--> access the [contacts]() section.